app/templates/main/success.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/panel/macro.html' import govukPanel -%}
+
+{% block beforeContent %}
+{{ super() }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <main id="main-content" role="main">
+        <section class="govuk-main-wrapper">
+            <div class="govuk-width-container">
+
+                {{ govukPanel({
+                "titleHtml": "Return submitted",
+                "text": file_name
+                }) }}
+
+                <p class="govuk-body">We have sent you a confirmation email.</p>
```

Comment:
Can we take this out for now until we're actually going to send a confirmation email?